src/lava/proc/io/sink.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import InPort
+from lava.magma.core.process.variable import Var
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyInPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires, tag
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+
+
+class ReceiveProcess(AbstractProcess):
+    """Receive process
```

Comment:
By the way is sink.py the best module name?

We have this dissonance in naming:
- package called 'io' meaning input/output
- module called sink
- class called Receive

input proc <-> sender <-> source
out proc <-> receiver <-> sink

Reply:
I am open to suggestions on the naming. it could well be `sink.BasicSink` or `reveive.BasicReceive`

Reply:
In general I'm voting for:
- package: io
- Different modules/packages for different types of input and output processes. The module/package names should be consistent with the class names found therein. If you call the module sink.py, don't call the process ReceiveProcess. In Particular there is no need to call it 'Process' since it is-a Process.

Before we assign too generic-sounding names for e.g. an Input process, we should think a bit on what type of input processes there will be in the future. You have a very particular type of input process that just injects a preprogrammed array of data over time. How should we call this versus one that loads images/arrays from a file? Gets it from a DVS camera, etc...